packages/mobile/src/common/enums/ui/date-time-picker-type.enum.ts

```diff
@@ -1,11 +1,13 @@
 enum DateTimeType {
   DATE = 'date',
   TIME = 'time',
+  DATE_TIME = 'datetime',
 }
 
 enum DateTimeFormat {
   DATE_ONLY = 'DD/MM/YYYY',
   TIME_ONLY = 'HH:mm',
+  DATE_TIME = 'DD/MM/YYYY HH:mm',
```

Comment:
Pls combine it under one enum with DateFormat

Reply:
corrected